app/views/partials/_header.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-expand-lg bg-body-tertiary  bg-dark border-bottom border-body" data-bs-theme="dark">
+  <div class="container-fluid">
+    <%= link_to "BlogApp", root_path, class:"navbar-brand"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <ul class="navbar-nav me-auto mb-2 mb-lg-0  nav justify-content-end" >
+      <li class="nav-item">
+        <%= link_to "Users", users_path, class:"nav-link"%>
+      </li>
```

Comment:
- [ OPTIONAL ] I think it would be great if you removed the user access before the user login or signup since to more authentication to not let the user be able to see the other users before signing up or logging in So I recommend removing it and if you want to display it you can display it after the user logging in ♻⭐💯


![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/35ad36ce-8bc4-4ee6-8f10-e46d09c57634)